announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/impl/SiteEmailNotificationAnnc.java

```diff
@@ -316,8 +316,13 @@ protected String getSubject(Event event)
 		String title = siteId;
 		try
 		{
-			Site site = siteService.getSite(siteId);
+			final Site site = siteService.getSite(siteId);
+			boolean shortDescription = ServerConfigurationService.getBoolean("sakai.announcement.email.subject.short.description", false);
+
 			title = site.getTitle();
+			if(shortDescription && site.getShortDescription() != null) {
```

Comment:
Would you mind to use StringUtils.isNotEmpty instead of "site.getShortDescription() != null" ?